multus/multus.go

```diff
@@ -402,11 +402,15 @@ func cmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) err
 		// https://github.com/kubernetes/kubernetes/issues/43014#issuecomment-287164444
 		_, ok := err.(ns.NSPathNotExistErr)
 		if ok {
-			return nil
+			logging.Debugf("cmdDel: WARNING netns may not exist, netns: %s, err: %s", netns, err)
```

Comment:
if there is no namespace, it should delegate the issue to the plugins ? Example Bridge plugin: https://github.com/containernetworking/plugins/blob/master/plugins/main/bridge/bridge.go#L508
Macvlan plugin: https://github.com/containernetworking/plugins/blob/master/plugins/main/macvlan/macvlan.go#L251

So the conclusion is the delegate plugins should take care of the issues, Multus should delegate it ?

Reply:
I think so, consider the case without Multus that the plugin is being called directly -- the plugin would have to handle this case just the same.

And in the case where the plugin has cleanup to do that's irrespective of the namespace being there or not,  it can still perform that cleanup.

Reply:
That is cool.. Agreed